pylib/gyp/msvs_emulation.py

```diff
@@ -935,7 +935,9 @@ def BuildCygwinBashCommandLine(self, args, path_to_base):
         )
         return cmd
 
-    RuleShellFlags = collections.namedtuple("RuleShellFlags", ["cygwin", "quote"])
+    class RuleShellFlags(typing.NamedTuple):
+        cygwin: int
```

Comment:
I believe that `mypy` would agree with this change but without running `mypy` in our GitHub Actions, I would be reluctant to merge this PR.
```suggestion
        cygwin: bool
```